### PR TITLE
Add support for subscription services

### PR DESCRIPTION
**Pull request by cyril-L**
I am posting this to get a review before actually passing homologation (most of the test cases get the expected result though).

I will do the homologation process after the review, to ensure the merged code actually pass. Could you wait for my confirmation before the merge?

### CODE REVIEW

File: doc/homologation.sh [hunk 1]
```diff
@@ -106,4 +152,4 @@ echo "F380-NR1 - Demande de transmission d’historique de courbe de charge dont
 lowatt-enedis cmdHisto --homologation $HOMOLOGATION_USER 25110853825340 $HOMOLOGATION_CONTRAT CDC --pas 30 --nom roro --civilite M --from 2010-01-01
 
 echo "F385B-NR1 - Demande de transmission d’historique d’index quotidiens d’un point avec une date de début à plus de 36 mois"
-lowatt-enedis cmdHisto --homologation $HOMOLOGATION_USER 25110853825340 $HOMOLOGATION_CONTRAT IDX --nom roro --civilite M --from 2010-01-01
+lowatt-enedis cmdHisto --homologation $HOMOLOGATION_USER 25110853825340 $HOMOLOGATION_CONTRAT IDX --nom roro --civilite M --from 2010-01-01
```
Comment (cyril-L) [inline doc/homologation.sh:161]: Oups, to be fixed

**Review by philpep — APPROVED** [review]
Hi @cyril-L , thanks for this !

I'm fine with the code, let us known when you've passed homologation and I'll be happy to merge this.

**Comment by cyril-L** [comment]
The code have been homologated, I am moving test cases to homologation.md.

**Comment by cyril-L** [comment]
Should be good now!

**Comment by philpep** [comment]
Merged, thanks a lot!

**Pull request merged**